code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5648,6 +5647,41 @@ var/procizine_tolerance = 0
 
 	M.bodytemperature += 10 * TEMPERATURE_DAMAGE_COEFFICIENT
 
+/datum/reagent/cold_ramen
+	name = "Cold Ramen"
+	id = COLD_RAMEN
+	description = "The tragic outcome of leaving hot ramen unfinished, but in the grand scheme of things, isn't all ramen cold?"
+	reagent_state = REAGENT_STATE_LIQUID
+	nutriment_factor = 5 * REAGENTS_METABOLISM
+	color = "#302000" //rgb: 48, 32, 0
+	density = 1.33
+	specheatcap = 4.18
+
+/datum/reagent/cold_ramen/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+
+	if(prob(4))
+		to_chat(M, "<span class='notice'>[pick("You are reminded of your school years, where cold ramen was a staple.","You long for a hot cup of ramen.")]</span>")
+
+/datum/reagent/frost_ramen
+	name = "Frost Ramen"
+	id = FROST_RAMEN
+	description = "Cold as ice ramen, the noodles are frozen."
+	reagent_state = REAGENT_STATE_SOLID
+	nutriment_factor = 5 * REAGENTS_METABOLISM
+	color = "#302000" //rgb: 48, 32, 0
+	density = 1.42
+	specheatcap = 14.59
+
+/datum/reagent/hell_ramen/on_mob_life(var/mob/living/M)
```

Comment:
you probably wanted to put frost ramen here, not hell ramen, seeing as you're reducing the bodytemp
also, temp adjusting with a multiplier of 10 is fucking insane
pure frost oil cools you down with a multiplier that scales from 0.3 to 0.9

did you test this? i'm pretty sure one sip of this is enough to send you unconscious and possibly even kill you outright

Reply:
That is the same exact scaling as hell ramen, literally just the sign changed.
One sip (5u) of hell ramen raises your temp to 340°C ish, and if you let it metabolize without dealing with the overheating, it can crit you and kill you from burn injuries: 
![image](https://github.com/vgstation-coders/vgstation13/assets/67024428/592bac8d-b22f-48ed-89be-66bc51576be4)
One sip (5u) of frost ramen lowers your temp to -313.7°C (kinda weird, once you go below absolute zero your guy stands up again, as he's no longer in the "freezing hypothermia" threshold). 
![image](https://github.com/vgstation-coders/vgstation13/assets/67024428/dbe5f4df-5cf4-4c7c-bf39-20d19578d1a5)
Of note is that you go unconscious almost instantly, as it only takes 1 tick to lose those 10°C required to go into severe hypothermia and thus faint, meanwhile pure frost oil takes about 25 ticks to send you into severe hypothermia.
It is about as deadly as hell ramen, but since you almost instantly faint, you can't do shit against it, it's essentially chloral. 
Would a scaling of -5 work? I still want the frozen ramen to be an opposite to hell ramen (in that it fucks your temperature so hard you can die from a single bite), but the hypothermia checks are so weak that even that feels a bit much.

Reply:
could gradually increase cooling rate each life tick like how frostoil does it to prevent instant knockout
and you don't need a high multiplier to kill people, frostoil's 0.9 is enough to stop your heart already
maybe slow the metabolism if you need 1 sip to be lethal

Reply:
I can't slow down the metabolism, it's supposed to be food first, lethal second. I'll tinker with the cooling rates so you faint at 20s, then it goes full force to cooling like it does now, so you have at least 20s to drink something warm, purge the thing or run to medical for help.

Reply:
Alright, it behaves like frost oil until the 20th tick, afterwards it behaves like it does currently, that way it is still deadly on one sip, but you have plenty of time to get warm or get help before you collapse.

Reply:
i'd personally just have it set your bodytemp to say -10C instead, so you can't get to negative temps
but this works too

also, you're still applying this to hellramen instead of frost ramen

Reply:
I'm looking into adding a check to the global temperature change handlers so they can't go under certain value, I'll set it so the minimum valid value is 0.15K, that way it is a good round number in celcius and stops the broken temperature handlers.

Reply:
also uhh those `max()`es will always set your bodytemp to 20C
you should have it `max(bodytemp-tempadjust, ABSOLUTE_ZERO)`

Reply:
that's literally copypasted from the frost oil code

Reply:
yeah i am retarded and my ESL kicked in
why aren't you using the `max()` at the last part, unless the intent is go under absolute zero and also husk the body

Reply:
It's just the hell ramen value copypasted as well but negative in sign. I think I found the correct temperature handle to add a sanity check, but I gotta test it so it doesn't break in unfathomable ways.